datafusion/functions-aggregate/src/approx_percentile_cont.rs

```diff
@@ -104,6 +105,12 @@ impl ApproxPercentileCont {
             None
         };
 
+        if args.ignore_nulls {
+            return not_impl_err!(
+                "IGNORE NULLS clause not yet supported for APPROX_PERCENTILE_CONT"
```

Comment:
Effectively this fix actually does ignore nulls, but we don't have a way to disable setting RESPECT NULLS.
I believe we should also move DataFusion to `SELECT percentile_cont(0.5) WITHIN GROUP(order by v)` (used by PostgreSQL, others) syntax...

Reply:
Maybe we can trigger it if used as a window function `PERCENTILE_CONT(...) OVER (PARTITION BY ... IGNORE NULLS)` 🤔

Reply:
Hm... for now I think it makes sense to ignore the `ignore_nulls` argument (like we did before) and remove any nulls which makes the most sense. Null handling is broken anyway (resulting in 0 instead of NULL), so no one will rely on percentile_cont on columns with nulls.
I'll file a ticket to discuss standardizing `PERCENTILE_CONT` and having it behave like PostgreSQL / Spark  / Snowflake, etc....

Reply:
https://github.com/apache/datafusion/issues/11732